src/components/application_manager/include/application_manager/message_helper.h

```diff
@@ -129,6 +129,17 @@ class MessageHelper {
   static hmi_apis::Common_Result::eType HMIResultFromString(
       const std::string& hmi_result);
 
+#ifdef SDL_REMOTE_CONTROL
+  /**
+   * @brief Converts string to device rank, if possible
+   * @param device_rank Stringified device rank
+   * @return Appropriate enum from device rank, or INVALID_ENUM, if conversion
+   * is not possible
+   */
+  static mobile_api::DeviceRank::eType StringToDeviceRank(
```

Comment:
I don't think we have DeviceRank of DRIVER or PASSENGER for RC baseline. Do we really need it?